README.md

```diff
@@ -141,7 +141,7 @@ Add to your `.babelrc`'s plugins array.
 ## Benchmarks
```

Comment:
It's best to try and keep commit comments to approximately 50 characters ([this post explains the reasoning](https://medium.com/@preslavrachev/what-s-with-the-50-72-rule-8a906f61f09c#.4xgaoohm9)). It just makes it easier to read in the various interfaces (e.g. GitHub, `git log`, etc.).